pallet/tests/test_utils.rs

```diff
@@ -0,0 +1,96 @@
+use eth_types::eth2::LightClientUpdate;
+
+use eth_types::{pallet::InitInput, BlockHeader};
+use lazy_static::lazy_static;
+
+pub fn read_headers(filename: String) -> Vec<BlockHeader> {
+    serde_json::from_reader(std::fs::File::open(std::path::Path::new(&filename)).unwrap()).unwrap()
+}
+
+pub fn read_client_update(filename: String) -> LightClientUpdate {
+    serde_json::from_reader(std::fs::File::open(std::path::Path::new(&filename)).unwrap()).unwrap()
+}
+
+pub fn read_client_updates(
+    network: String,
+    start_period: u64,
+    end_period: u64,
+) -> Vec<LightClientUpdate> {
+    let mut updates = vec![];
+    for period_idx in start_period..=end_period {
+        let client_update = read_client_update(format!(
+            "./tests/data/{network}/light_client_update_period_{period_idx}.json"
+        ));
+        updates.push(client_update);
+    }
+
+    updates
+}
+
+pub struct InitOptions<AccountId> {
+    pub validate_updates: bool,
+    pub verify_bls_signatures: bool,
+    pub hashes_gc_threshold: u64,
+    pub trusted_signer: Option<AccountId>,
+}
+
+pub fn get_goerli_test_data(
+    init_options: Option<InitOptions<[u8; 32]>>,
+) -> (
+    &'static Vec<Vec<BlockHeader>>,
+    &'static Vec<LightClientUpdate>,
+    InitInput<[u8; 32]>,
+) {
+    const NETWORK: &str = "goerli";
+    lazy_static! {
+        static ref INIT_UPDATE: LightClientUpdate =
+            read_client_updates(NETWORK.to_string(), 632, 632)[0].clone();
+        static ref UPDATES: Vec<LightClientUpdate> =
+            read_client_updates(NETWORK.to_string(), 633, 633);
+        static ref HEADERS: Vec<Vec<BlockHeader>> = vec![read_headers(format!(
+            "./tests/data/{}/execution_blocks_{}_{}.json",
+            NETWORK, 8652100, 8661554
+        )),];
```

Comment:
you can use [OnceCell](https://doc.rust-lang.org/stable/std/cell/struct.OnceCell.html) here instead of `lazy_static`

Reply:
It was copy paste, but good catch. Replaced it.
Done